src/Microsoft.Sbom.Extensions.DependencyInjection/RemapComponentDetectionErrorsToWarningsLogger.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom;
+
+using System;
+using Serilog;
+using Serilog.Events;
+
+/// <summary>
+/// A class to remap Errors logged from ComponentDetection assemblies to Warnings.
+/// </summary>
+public class RemapComponentDetectionErrorsToWarningsLogger : ILogger
```

Comment:
Much of this codebase doesn't have any unit tests, but IMO we should be trying to fix that as we add new classes as much as possible/is reasonable. Would you be able to add unit tests for this class?

Reply:
Unit tests have been added